wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -877,6 +883,10 @@ public final class SessionManager: NSObject, SessionManagerType {
         }
     }
 
+    public func enqueueRetryStart() {
```

Comment:
does it really enqueue here ?

```suggestion
    public func retryStart() {
```